internal/kokoro/test.sh

```diff
@@ -29,5 +29,17 @@ try3() { eval "$*" || eval "$*" || eval "$*"; }
 try3 go mod download
 ./internal/kokoro/vet.sh
 
+go get github.com/jstemmer/go-junit-report
+
 # Run tests and tee output to log file, to be pushed to GCS as artifact.
-go test -race -v ./... 2>&1 | tee $KOKORO_ARTIFACTS_DIR/$KOKORO_GERRIT_CHANGE_NUMBER.txt
+go test -race -v ./... 2>&1 | tee $KOKORO_ARTIFACTS_DIR/sponge_log.log
```

Comment:
`set +e`

Reply:
I think we needed to do this gocloud so that failures got reported?

Reply:
Good catch. Done.